_projects/2020/juliaastro/ccd_data_reduction.md

```diff
@@ -0,0 +1,32 @@
+---
+name: CCD Processing Package
+desc: Develop a CCD data reduction package in Julia
+# add a short one line description of your project
+requirements:
+# Student requirements:
+ - Basic knowledge of CCD data
+ - Familiarity with Julia and Julia packaging
+difficulty: low
+issues:
+# Related issues (if any)  to this project.
+mentors:
+# First person in contact; mentors may change before project starts.
+# GitHub handles
+ - mileslucas
```

Comment:
We have a requirement to have at least two mentors per projects. Could you please add someone else agree to co-mentor?

Reply:
no problem- I'll find another